beacon-chain/core/epoch/epoch_processing.go

```diff
@@ -24,9 +24,9 @@ import (
 // MatchedAttestations is an object that contains the correctly
 // voted attestations based on source, target and head criteria.
 type MatchedAttestations struct {
-	source []*pb.PendingAttestation
-	target []*pb.PendingAttestation
-	head   []*pb.PendingAttestation
+	Source []*pb.PendingAttestation
```

Comment:
why does this need to be exported, they are all used in the same package

Reply:
You are right, they don't have be. Fixed